api/src/app.py

```diff
@@ -52,6 +53,17 @@ class EndpointConfig(PydanticBaseEnvConfig):
     auth_endpoint: bool = Field(False, alias="ENABLE_AUTH_ENDPOINT")
 
     enable_apply_endpoints: bool = Field(False, alias="ENABLE_APPLY_ENDPOINTS")
+    domain_verification_content: str | None = Field(None, alias="DOMAIN_VERIFICATION_CONTENT")
+    domain_verification_map: dict | None = None
+
+    def model_post_init(self, _context: Any) -> None:
+        self.domain_verification_map = {}
```

Comment:
What if we just do this - make the value never None, which cleans up a few of the type checks you do with it.
```suggestion
    domain_verification_map: dict = Field(default_factory=dict)

    def model_post_init(self, _context: Any) -> None:
```